curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/655209a4a27dd37873c4cac3.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 655209a4a27dd37873c4cac3
+title: Step 14
+challengeType: 20
+dashedName: step-14
+---
+
+# --description--
+
+`find()` returns the index of the character inside the parsed string. If the character is not found, it returns `-1`. As you can see, the first character in `text`, uppercase "H", is not found, since `alphabet` contains lowercase letters only.
```

Comment:
```suggestion
`find()` returns the index of the matching character inside the string. If the character is not found, it returns `-1`. As you can see, the first character in `text`, uppercase `"H"`, is not found, since `alphabet` contains only lowercase letters.
```